core/src/main/scala/caliban/wrappers/Wrapper.scala

```diff
@@ -41,6 +41,16 @@ object Wrapper {
     def wrap[R1 <: R](f: Info => ZIO[R1, E, A]): Info => ZIO[R1, E, A]
   }
 
+  /**
+   * A wrapper that doesn't do anything.
+   * Useful for cases where we want to programmatically decide whether we'll use a wrapper or not
+   */
+  val empty: Wrapper[Any] = new OverallWrapper[Any] {
```

Comment:
Override `|+|` to return only the right part?

Reply:
I ended up creating an object for it which overrides both `|+|` and `apply`. Let me know what you think.

On that note, a bit more philosophical question: should it be called `Empty` or something like `Id` / `Identity`?

Reply:
I like `Empty`, I used the same pattern here: https://github.com/ghostdogpr/caliban/blob/58c0d76fcc8ec894e5ffc727d59511dc343a425e/core/src/main/scala/caliban/introspection/adt/__Type.scala#L164

Reply:
it makes `toString` nicer 😄 I don't think there's any other benefit

Reply:
Looking at the implementations of `Nil`, `None` as well as your `TypeVisitor.Empty`, they all seem to be `case object`s so I'll use that instead

Reply:
https://scastie.scala-lang.org/S8QMz72CRlWJQiBof5b0fw

Reply:
Not super critical but sometimes when debugging it helps a bit.

Reply:
Another thing I noticed is that it makes the `.hashCode()` method consistent when executing the same program multiple times. I'm not sure whether this has any real value though